.gitlab/ci/on-push.yml

```
@@ -88,11 +88,64 @@ create-instances:
       [ -n "${TIME_TO_LIVE}" ] && TTL=${TIME_TO_LIVE} || TTL=300
       python3 ./Tests/scripts/awsinstancetool/aws_instance_tool.py -envType "$IFRA_ENV_TYPE" -timetolive $TTL -outfile "$ARTIFACTS_FOLDER/env_results.json"
     - section_end "Create Instances"
-    - section_start "Upload Artifacts to GCP"
+    - section_start "Upload Artifacts to GCP" --collapsed
     - ./Tests/scripts/upload_artifacts.sh
     - section_end "Upload Artifacts to GCP"
 
 
+create-instances-mpv2:
+  tags:
+    - gke
+  extends:
+    - .default-job-settings
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/'
+    - if: '$NIGHTLY'
+    - if: '$INSTANCE_TESTS'
+  cache:
+    policy: pull-push
+  needs: []
+  stage: create-instances
+  script:
+    - !reference [.download-demisto-conf]
+    - !reference [.create-id-set-mp-v2]
+    - section_start "Create Release Notes and Common Server Documentation" --collapsed
+    - echo "Creating Release Notes and Content Descriptor"
+    - python3 Utils/release_notes_generator.py $CONTENT_VERSION $GIT_SHA1 $CI_BUILD_ID --output $ARTIFACTS_FOLDER/packs-release-notes.md --github-token $GITHUB_TOKEN
+    - cp content-descriptor.json $ARTIFACTS_FOLDER
+    - ./Documentation/commonServerDocs.sh
+    - section_end "Create Release Notes and Common Server Documentation"
+
+    - section_start "Create Content Artifacts and Update Conf" --collapsed
+    - demisto-sdk create-content-artifacts -a $ARTIFACTS_FOLDER --cpus 8 --content_version $CONTENT_VERSION --marketplace "marketplacev2" >> $ARTIFACTS_FOLDER/logs/create_content_artifacts.log
```

```suggestion
    - demisto-sdk create-content-artifacts -a $ARTIFACTS_FOLDER --cpus 8 --content_version $CONTENT_VERSION --marketplace "marketplacev2" -af -fbi -idp ./Tests/id_set_mp_v2.json>> $ARTIFACTS_FOLDER/logs/create_content_artifacts.log
```

adding the filtering by id_set (-fbi) + alternating fields (-af)

---

also this should be coordinated with reut's pr's changes, if there will be any